processing/src/main/java/io/druid/query/groupby/RowBasedColumnSelectorFactory.java

```diff
@@ -410,19 +418,25 @@ public void inspectRuntimeShape(RuntimeShapeInspector inspector)
         @Override
         public double getDouble()
         {
-          return row.get().getMetric(columnName).doubleValue();
+          return DimensionHandlerUtils.nullToZeroDouble(row.get().getMetric(columnName)).doubleValue();
+        }
+
+        @Override
+        public boolean isNull()
```

Comment:
Please move this method either before or after all getXxx() methods

Reply:
done.